### PR TITLE
[prometheus] wrote the username for basic auth

### PR DESCRIPTION
**Pull request by alexey-gavrilov-flant**
## Description
Minor documentation updates.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: prometheus
type: fix 
summary: Minor documentation updates.
impact_level: low
```

### CODE REVIEW

File: modules/300-prometheus/docs/CONFIGURATION.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The module does not require any configuration – it works right out-of-the-box.
 ## Authentication
 
 [user-authn](/documentation/v1/modules/150-user-authn/) module provides authentication by default. Also, externalAuthentication can be configured (see below).
-If these options are disabled, the module will use basic auth with the auto-generated password.
+If these options are disabled, the module will use basic auth with the auto-generated password and user `admin`.
```
Comment (z9r5) [inline modules/300-prometheus/docs/CONFIGURATION.md:15]:
```suggestion
If these options are disabled, the module will use basic auth with the auto-generated password and the user `admin`.
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [7701](https://github.com/deckhouse/deckhouse/pull/7701) to the branch [release-1.58](https://github.com/deckhouse/deckhouse/tree/release-1.58) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [7702](https://github.com/deckhouse/deckhouse/pull/7702) to the branch [release-1.57](https://github.com/deckhouse/deckhouse/tree/release-1.57) successful!

**Pull request merged**